com.unity.netcode.gameobjects/Runtime/com.unity.netcode.runtime.asmdef

```diff
@@ -28,6 +28,9 @@
             "name": "com.unity.multiplayer.tools",
             "expression": "1.0.0-pre.4",
             "define": "MULTIPLAYER_TOOLS_1_0_0_PRE_4"
+        },
+        {
+            "name": "com.unity.multiplayer.tools",
+            "expression": "1.0.0-pre.7",
+            "define": "MULTIPLAYER_TOOLS_1_0_0_PRE_7"
```

Comment:
I wonder if we should just change all of these to _PRE_7 to simplify? We don't actually need to support both versions, these stats aren't useful until RNSM comes out so even though technically pre.6 will have the metric ids they don't need to actually work yet.

Reply:
Sure, why not. I see no harm.